features/onboarding/src/main/java/com/solid/ufc/onboarding/view/OnboardingScreen.kt

```diff
@@ -1,73 +1,192 @@
 ) {
-    LaunchedEffect("goToHome") {
+
+    val items = listOf(
+        PagerItemData(
+            title= stringResource(R.string.pager_one_title),
+            description=stringResource(R.string.pager_one_desc),
+            image = painterResource(R.drawable.onb1)
+    ),     PagerItemData(
+            title= stringResource(R.string.pager_two_title),
+            description=stringResource(R.string.pager_two_desc),
+            image = painterResource(R.drawable.onb2)
+    ),     PagerItemData(
+            title= stringResource(R.string.pager_three_title),
+            description=stringResource(R.string.pager_three_desc),
+            image = painterResource(R.drawable.onb3)
+    ),
+    )
+    val pagerState = rememberPagerState(pageCount = {
+        items.size
+    })
+    val coroutineScope = rememberCoroutineScope()
+//    LaunchedEffect("goToHome") {
 //        viewModel.goToHome(rootNavController) //you can comment this out when you want to implement going to the home screen
-    }
+//    }
+
+     Scaffold (
+         topBar = {
+             Row(
+                 modifier = Modifier
+                     .fillMaxWidth()
+                     .fourPadding(),
+                 horizontalArrangement = Arrangement.SpaceBetween
+             ) {
+                 Spacer(Modifier.height(3.dp))
 
-    Column(
-        modifier = Modifier
-            .fillMaxSize()
-            .background(MaterialTheme.colorScheme.background)
-    ) {
-        Card(
-            modifier = Modifier
-                .onePadding()
-                .fillMaxSize(),
-            colors = PrimaryCardColors.copy(containerColor = MaterialTheme.colorScheme.background),
+                 Text(text = stringResource(R.string.skip))
+             }
+         },
+bottomBar = {
 
-            ) {
-            Column(
-                horizontalAlignment = Alignment.CenterHorizontally
-            ) {
+   Row(
+       horizontalArrangement = Arrangement.Center,
+       modifier = Modifier
+           .fillMaxWidth()
+           .padding(bottom = 70.dp)
+   ){
+       Box(
+           contentAlignment = Alignment.Center,
+           modifier = Modifier
+               .size(66.dp)
+               .background(
+                   color = Color(0xFF446EA6),
+                   shape = CircleShape
+               )
+               .clickable {
+                   if(pagerState.currentPage ==items.size-1){
 
+                   }else{
+                      coroutineScope.launch {
+                          pagerState.animateScrollToPage(pagerState.currentPage+1)
+                      }
+                   }
+               }
+       ) {
+           if(pagerState.currentPage !=items.size-1 ){
+               Image(
+                   painter =painterResource(R.drawable.arrow_back),
+                   contentDescription = null,
+                   contentScale = ContentScale.Crop,
+                   modifier = Modifier.size(24.dp)
```

Comment:
Great work man, just some few cleanup here: 
Change Modifier.size(24.dp) to Modifier.size(SpacingTheme.three)